modules/context/api.go

```diff
@@ -77,6 +77,10 @@ func (ctx *APIContext) Error(status int, title string, obj interface{}) {
 
 	if status == http.StatusInternalServerError {
 		log.ErrorWithSkip(1, "%s: %s", title, message)
+
+		if macaron.Env == macaron.PROD {
+			message = ""
```

Comment:
we may should leave a hint to enable DEV for debugging?

Reply:
Meh. I mean it'll be the same as the other ISEs